docs/user/cli/commands/kyverno_create_permission.md

```diff
@@ -0,0 +1,45 @@
+## kyverno create permission
+
+Create an aggregated role for a given resource type
+
+### Synopsis
+
+This command generates a Kubernetes ClusterRole and ClusterRoleBinding for a specified resource type.
+The generated files will be saved in the user's home directory under the "aggregated-role" folder.
```

Comment:
Can we simply output on `stdout` so the command can be used in a pipe with `kubectl apply`?